migration_tools.drush.inc

```diff
@@ -46,6 +46,21 @@ function migration_tools_drush_command() {
     ),
   );
 
+  $items['migration-tools-html-file-list'] = array(
+    'description' => "List all HTML files in a specified directory.",
+    'arguments' => array(
+      'directory' => 'Path to the directory to analyze.',
+    ),
+    'options' => array(
+      'outfile' => 'Write output to a file',
+      'format' => 'Specify output format (options: csv).',
+    ),
+    'examples' => array(
+      'drush migration-tools-html-file-list ~/old_site/' => 'Lists HTML files found in of ~/old-site and all subdirectories.',
+      'drush migration-tools-html-file-list ~/old_site --format=csv --outfile=results.csv' => 'Lists HTML files in the top-level of ~/old-site in CSV format, saved to a file under the document root.',
```

Comment:
I like all the new changes.   This example needs to be updated to remove the outfile option.